graphics.cpp

```diff
@@ -1,17 +1,10 @@
 #include "graphics.h"
 #include "resources.h"
 
-// Here, textures are born not by loading from disk...
```

Comment:
Unfortunately I had to kill this poem because part of it had to get moved to another file.